operations/operations/company-policies/community-response-policy.md

```diff
@@ -0,0 +1,69 @@
+# Community Response Policy
+
+Sensitive topics listed below are escalated to a [Community Escalations channel](https://community.mattermost.com/private-core/channels/community-escalations) in the Private Staff team, which includes all members of the Mattermost Leadership Team (MLT) plus any approved responders. This channel is public, so anyone in the company can escalate an issue. 
+
+When an issue relating to a sensitive topic is raised (for example, on GitHub issues, Twitter, or elsewhere), it should be posted in the Community Escalations channel with an @-mention to the list of approved responders related to that particular issue.
+
+## Escalation Process
+
+Raising an issue is the responsibility of whoever is managing community responses in a given forum. However, it is also open for anyone at the company to escalate an issue that they see.
+
+It is then the responsibility of the approved responders to review the issue and choose an owner for responding (if a response is appropriate). By default, ownership is delegated to the first person on the list of approved responders. 
+
+## Sensitive Topics
```

Comment:
@justinegeffen You suggested title case for this heading, but not the subheadings below. Can you help remind me what the rule with headings is?